EmbeddedDocumentsQuery/src/EmbeddedDocumentsQuery.java

```diff
@@ -0,0 +1,127 @@
+import java.util.ArrayList;
+import java.util.List;
+
+import org.bson.Document;
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+//import com.mongodb.client.model.Projections;
+
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Filters.and;
+import static com.mongodb.client.model.Filters.gt;
+//import static com.mongodb.client.model.Filters.lt;
+//import static com.mongodb.client.model.Filters.gte;
+//import static com.mongodb.client.model.Filters.lte;
+//import static com.mongodb.client.model.Projections.fields;
+//import static com.mongodb.client.model.Projections.include;
+//import static com.mongodb.client.model.Projections.excludeId;
+
+public class EmbeddedDocumentsQuery {
+	
+	/**
+	 * Retrieves documents filtered by shipping address - city.
+	 * @param connectionString: To MongoDB instance/MongoDB Cluster.
+	 * @param city: Shipping address - city to filter documents.
+	 * @return List<Document>: ArrayList of matching documents.
+	 */
+	
+	public List<Document> SingleNestedFieldEqualityMatch(String connectionString, String city){
```

Comment:
As shannon mentioned, the method names should reflect what the method does (example: filterByCity) rather than the operation it does